lib/generators/active_admin/webpacker/webpacker_generator.rb

```diff
@@ -0,0 +1,26 @@
+module ActiveAdmin
+  module Generators
+    class WebpackerGenerator < Rails::Generators::Base
+
+      source_root File.expand_path('templates', __dir__)
+
+      def install_assets
+        template 'active_admin.js', 'app/javascript/packs/active_admin.js'
+        template "active_admin.scss", "app/javascript/stylesheets/active_admin.scss"
+        template 'print.scss', 'app/javascript/packs/active_admin/print.scss'
+
+        copy_file "#{__dir__}/plugins/jquery.js", Rails.root.join("config/webpack/plugins/jquery.js").to_s
+
```

Comment:
I think the last commit lost some stuff that's important to get tests green. At least I need something like

```diff
+        gsub_file Rails.root.join("config/webpacker.yml"), /^(.*)extract_css.*$/, '\1extract_css: true'
+
```

here to get some scenarios to pass (the ones about asset registration).

Reply:
Added for tests only as it's the only way to get `css` files and check for them to be present. Not necessary for development app